src/main/java/edu/nicholaidudakiwwarrick/advancedjava/app/StockQuoteApplication.java

```diff
@@ -0,0 +1,122 @@
+package edu.nicholaidudakiwwarrick.advancedjava.app;
+
+import edu.nicholaidudakiwwarrick.advancedjava.model.StockQuote;
+import edu.nicholaidudakiwwarrick.advancedjava.services.ServiceFactory;
+import edu.nicholaidudakiwwarrick.advancedjava.services.ServiceType;
+import edu.nicholaidudakiwwarrick.advancedjava.services.StockService;
+import edu.nicholaidudakiwwarrick.advancedjava.services.StockServiceException;
+import edu.nicholaidudakiwwarrick.advancedjava.util.IntervalEnum;
+import org.joda.time.DateTime;
+import org.joda.time.format.DateTimeFormatter;
+
+import javax.validation.constraints.NotNull;
+import java.util.List;
+
+/**
+ * Basic class with a Main method to create StockService instances
+ */
+public class StockQuoteApplication {
+    /**
+     * @param args an array that contains elements:
+     *             <code>String</code> representation of stock symbol, start date, and end date
+     */
+    public static void main(@NotNull String[] args) {
+
+        StockService stockService = null;
+
+        // create a StockService instance using the ServiceFactory static factory method
+        try {
+            stockService = ServiceFactory.getStockServiceInstance(ServiceType.BASIC);
+        } catch (StockServiceException e) {
+            e.getMessage();
```

Comment:
this is not good exception handling. exceptions should be handled not just printed to console.